.github/workflows/build_pr_preview.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
       - name: Build and minify using Hugo
         # Build in a directory specific to the PR number
-        run: hugo --minify --destination public/${{ github.event.number }}
+        run: hugo -D --minify --destination public/${{ github.event.number }}
```

Comment:
```suggestion
        run: hugo --buildDrafts --minify --destination public/${{ github.event.number }}
```

Reply:
Still not willing to support this, I would prefer to have the preview really look like the final output.
And people can just commit without the draft setting and have the preview work.